prov/gni/src/gnix_rma.c

```diff
@@ -881,6 +1082,8 @@ ssize_t _gnix_rma(struct gnix_fid_ep *ep, enum gnix_fab_req_type fr_type,
 	int rc;
 	int rdma;
 	struct fid_mr *auto_mr = NULL;
+	struct gnix_fab_req *more_req;
+	struct slist_entry *sle;
 
```

Comment:
@ztiffany had suggested letting operation continue as normal - as it does now if someone uses the FI_MORE flag. I'm fine with either approach.